### PR TITLE
Refactor verifier.go to use backchannel for responses

### PR DESCRIPTION
**Pull request by leroxyl**
https://ubirch.atlassian.net/browse/UP-1220

### CODE REVIEW

File: main/udp_server.go [hunk 1]
```diff
@@ -61,11 +63,45 @@ func (srv *UDPServer) Listen(addr string, ctx context.Context, wg *sync.WaitGrou
...
+			srv.receiveHandler <- buffer[:n]
 		}
 	}()
 
 	log.Printf("UDP server up and listening on %v", udpAddr)
 
 	return nil
 }
+
+//noinspection GoUnhandledErrorResult
```
Comment (thinkberg) [inline main/udp_server.go]: ist das gut?

File: main/udp_server.go [hunk 2]
```diff
@@ -61,11 +63,45 @@ func (srv *UDPServer) Listen(addr string, ctx context.Context, wg *sync.WaitGrou
...
+	if err != nil {
+		return err
+	}
+	connection, err := net.DialUDP("udp", nil, udpAddr)
+	if err != nil {
+		panic(fmt.Sprintf("network error setting up response sender: %v", err))
+	}
+
+	go func() {
+		defer wg.Done()
```
Comment (thinkberg) [inline main/udp_server.go:85]: Wo wird denn die wg +1 gemacht fuer dieses Done?

File: main/udp_server.go (hunk 1 above)
Comment (leroxyl) [reply main/udp_server.go]: probably not. taking it out.

File: main/udp_server.go (hunk 2 above)
Comment (leroxyl) [reply main/udp_server.go:85]: https://github.com/ubirch/ubirch-go-udp-client/blob/UP-1220/main/main.go#L148

**Pull request merged**